<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>805</width>
    <height>136</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Credentials File</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="credential_file_edit_line"/>
     </item>
     <item>
      <widget class="QPushButton" name="get_credential_file_button">
       <property name="text">
        <string>Get File</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QPushButton" name="check_connection_button">
       <property name="text">
        <string>Check Connection</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="status_edit_line"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Shared Folder</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="shared_folder_edit_line"/>
     </item>
     <item>
      <widget class="QPushButton" name="get_shared_folder_button">
       <property name="text">
        <string>Get Folder</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Results Folder</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="results_folder_eddit_line"/>
     </item>
     <item>
      <widget class="QPushButton" name="get_results_folder_button">
       <property name="text">
        <string>Get Folder</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
